lib/views/home_content/feed/post_card/post_header_widget.dart

```diff
@@ -0,0 +1,72 @@
+import "package:cloud_firestore/cloud_firestore.dart";
+import "package:flutter/material.dart";
+import "package:intl/intl.dart";
+import "package:proxima/utils/ui/user_avatar.dart";
+import "package:timeago/timeago.dart";
```

Comment:
Ok, I didn't think it was mandatory.
Fixed in e770ca0493e8393870fd9c7b32040ebbc5e4911d

Reply:
It is not mandatory, but it is better to follow the official documentation.
Especially in that case where otherwise you just call this `format` method, and we don't really know where it comes from.